EDAM_dev.owl

```diff
@@ -59834,11 +59834,13 @@
     <owl:Class rdf:about="http://edamontology.org/topic_3523">
         <rdfs:subClassOf rdf:resource="http://edamontology.org/topic_3077"/>
         <created_in>1.8</created_in>
-        <oboInOwl:hasDefinition>RNAi experiments.</oboInOwl:hasDefinition>
+        <oboInOwl:hasDefinition>Experiment in which the expression of a specific gene is post-transcriptionally knocked down or silenced by means of a small RNA (sRNA) causing the degradation of the gene transcript (mRNA), and thus preventing the production of the corresponding protein.</oboInOwl:hasDefinition>
         <oboInOwl:hasHumanReadableId>RNAi_experiment</oboInOwl:hasHumanReadableId>
         <oboInOwl:inSubset rdf:resource="http://edamontology.org/bio"/>
         <oboInOwl:inSubset rdf:resource="http://edamontology.org/topics"/>
-        <rdfs:label>RNAi experiment</rdfs:label>
+        <oboInOwl:savedBy>Claire Rioualen</oboInOwl:savedBy>
+        <rdfs:label rdf:datatype="http://www.w3.org/2001/XMLSchema#string">RNAi experiment</rdfs:label>
+        <rdfs:seeAlso rdf:resource="https://en.wikipedia.org/wiki/Post-transcriptional_regulation"/>
         <rdfs:seeAlso rdf:resource="https://en.wikipedia.org/wiki/RNA_interference"/>
```

Comment:
This [RNAi](https://en.wikipedia.org/wiki/RNA_interference) itself is also described more as a biological phenomenon, so it would be good to include an according skos:relatedMatch.

And we should talk with experts to review the way we model these as data acquisition/experiment techniques. A good example might be https://en.wikipedia.org/wiki/CRISPR_interference

Reply:
And a similar skos:relatedMatch to http://purl.bioontology.org/ontology/MESH/D034622